resolversrv/views.py

```diff
@@ -474,3 +532,17 @@ def resolver(bibcode, link_type):
     :return:
     """
     return LinkRequest(bibcode, link_type.upper(), current_app.config['RESOLVER_GATEWAY_URL']).process_request()
+
+@advertise(scopes=[], rate_limit=[1000, 3600 * 24])
+@bp.route('/update', methods=['POST'])
+def update():
+    """
+    """
+    try:
+        payload = request.get_json(force=True)  # post data in json
+    except:
+        payload = dict(request.form)  # post data in form encoding
+
+    print '............'
```

Comment:
you can replace it with `current_app.logger.debug('Received payload: %s', payload)`

Reply:
sorry, this was a debug print we added when Steve and I were testing.